doc/source/whatsnew/v0.24.1.rst

```diff
@@ -66,7 +66,7 @@ Bug Fixes
 **Reshaping**
 
 - Bug in :func:`merge` when merging by index name would sometimes result in an incorrectly numbered index (:issue:`24212`)
-
+Bug in :meth:`DataFrame.groupby`(:class:`pd.Grouper`(freq='1d')) when there is a time change (DST) and grouping frequency is 1d (:issue:`24972`)
```

Comment:
I don't think this will render the way you intend it to (parenthesis and `freq=` won't be in code style).  I think it'd be clearer to separate the `DataFrame.groupby` and `Grouper`, so that they just reference the method/class (no params passed), and then stylize the "1d" at the end.  Something like:

```rst
Bug in :meth:`DataFrame.groupby` with :class:`Grouper` ... frequency is ``'1d'`` (:issue:`24972`)
```

Reply:
updated

Reply:
thanks!